src/components/Stepper/Step/Step.scss

```diff
@@ -0,0 +1,115 @@
+@import "vanilla-framework";
+
+.p-list__item {
+  padding-bottom: 0;
+  padding-top: 0;
+}
+
+.p-inline-list {
+  display: flex;
+}
```

Comment:
Those look like standard vanilla classes, which we shouldn't override globally. 

I think best is 
1. to namespace all selectors in this file with a wrapper and 
2. target only custom classes

Reply:
Agreed, I've shifted the css into the `stepper-vertical` and `stepper-horizontal` classes.